contents/docs/libraries/react-native/_snippets/identify.mdx

```diff
@@ -3,17 +3,29 @@ Using `identify`, you can associate events with specific users. This enables you
 An `identify` call has the following arguments:
 
 - **distinctId:** Required. A unique identifier for your user. Typically either their email or database ID.
-- **userProperties:** Optional. A dictionary with key:value pairs to set the [person properties](/docs/product-analytics/person-properties)
```

Comment:
is it better to refer to them as person properties consistently, rather than user properties?

not blocking though 😊

Reply:
`properties` is the name of the method parameter (method signature).
I removed the userProperties and userPropertiesSetOnce comments since they don't make any sense to flutter/node anyway, it was likely a copy-paste from the other SDKs where the method signature have those as optional parameters for $set and $set_once